doc.ddoc

```diff
@@ -66,27 +66,54 @@ $(DIVID cssmenu, $(UL
         http://twitter.com/search?q=%23dlang, Twitter,
         http://digitalmars.com/d/dlinks.html, More Links
       )
+    $(MENU_W_SUBMENU Compilers $(AMP) Tools)
+      $(SUBMENU
+        dmd-windows.html, dmd $(NDASH) reference compiler,
+        http://gdcproject.org, gdc $(NDASH) gcc-based compiler,
+        http://wiki.dlang.org/LDC, ldc $(NDASH) LLVM-based compiler,
+        rdmd.html, rdmd $(NDASH) pseudo interpreter,
```

Comment:
rdmd is more important as a build tool than a pseudo interpreter (for which `dmd -run` is a closer fit).